conf/hadoop-metrics2-hbase.properties

```diff
@@ -14,31 +14,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# syntax: [prefix].[source|sink].[instance].[options]
-# See javadoc of package-info.java for org.apache.hadoop.metrics2 for details
+## syntax: [prefix].[source|sink].[instance].[options]
+## See javadoc of package-info.java for org.apache.hadoop.metrics2 for details
 
-*.sink.file*.class=org.apache.hadoop.metrics2.sink.FileSink
-# default sampling period
+## Default sampling period, in seconds
```

Comment:
Used double `#` to distinguish actual comments from commented-out configuration lines.